Source/Scene/Cesium3DTile.js

```diff
@@ -124,6 +124,20 @@ define([
         /**
          * DOC_TBA
          *
+         * @readonly
+         */
+        this.hasSubtree = false;
```

Comment:
The `hasSubtree` and `subtreeLoading` terminology may be confusing since all non-leaf tiles have sub-trees.  Maybe these should be `isTileset` and `tilesetLoading`?